docs/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- Is there a Trello card to link to? -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Prison details update (e.g. slot updates)
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask! -->
+- [ ] My code follows the code style of this project.
```

Comment:
I wonder if the code style bit is necessary as Code Climate will not pass the build unless styles are adhered to?

Reply:
@TobyRet - Agreed!  Thought we would probably get rid of this, think I just left it in as a way of showing what kinds of checks we could use.